pkg/api/vlabs/const.go

```diff
@@ -67,3 +71,8 @@ const (
 	// ManagedDisks means that the nodes use managed disks for their os and attached volumes
 	ManagedDisks = "ManagedDisks"
 )
+
+// Network policy
+var (
+	NetworkPolicyValues = [...]string{"", "none", "azure", "calico"}
```

Comment:
I guess "" is when there are no values specified. "none" is what acs-engine internally uses. In any case, those values were not introduced nor changed by this PR. I agree it would be nice to clean that up later.